me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/IDmeWebVerify.java

```diff
@@ -243,9 +254,19 @@ private void SendDataBack(String response)
     {
         activity.getIntent().putExtra(IDME_WEB_VERIFY_RESPONSE, response);
         activity.setResult(Activity.RESULT_OK, activity.getIntent());
-
         activity.finish();
     }
 
-
+    /**
+     * Sends the error back to the activity that called the WebView.
+     *
+     * @param error The service error form the Web Request.
+     */
+    private void SendErrorBack(String error)
```

Comment:
currently the communication between activities is via intents, we can try serialize an exception and send it. However, this communication way will change with the refresh token feature. So, in my opinion we have to improve it in another pull. makes sense?